### PR TITLE
Improve README.md

### LINKED ISSUE #5: Improve repo readme
Add longer description, general tidying for summer school applicability.

### PR DESCRIPTION
**Pull request by MarionBWeinzierl**
Closes #5 . 

Add a longer description and some additional info to the repo readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # power-up-python
```
Comment (jatkinson1000) [inline README.md]: Perhaps rename to RSE Skills or similar to reflect summer school title?
```suggestion
<img src="https://iccs.cam.ac.uk/sites/iccs.cam.ac.uk/files/logo2_1.png"  width="355" align="left">

<br><br><br><br><br>

# ICCS RSE Skills (in Python) Workshop
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 
-Materials for a workshop for educating academic researchers in research software
-engineering (RSE) principles.
+Materials (slides and exercises) for a workshop for educating academic researchers in research software engineering (RSE) principles. The examples shown are geared towards climate scientists, 
+but the material is generally suited for people with various backgrounds.
```
Comment (jatkinson1000) [inline README.md:8]: Yes, I like making this point! 😃

**Review by jatkinson1000 — APPROVED** [review]
Thanks @MarionBWeinzierl

Everything here looks good.
I'm just looking at the JOSE guidelines wonder if we could add anything else like learning objectives.

We could also consider adding installation instructions, license badges, and authors (I'm taking inspiration from https://github.com/Cambridge-ICCS/practical-ml-with-pytorch)

**Comment by MarionBWeinzierl** [comment]
> Everything here looks good. I'm just looking at the JOSE guidelines wonder if we could add anything else like learning objectives.
> 
> We could also consider adding installation instructions, license badges, and authors (I'm taking inspiration from https://github.com/Cambridge-ICCS/practical-ml-with-pytorch)

I added issue #12 for this, but will merge the current changes in now.

**Pull request merged**